geth/common/types.go

```diff
@@ -436,18 +436,24 @@ type NotifyResult struct {
 const passphraseEnvName = "ACCOUNT_PASSWORD"
 
 // LoadTestConfig loads test configuration values from disk
-func LoadTestConfig() (*TestConfig, error) {
+func LoadTestConfig(networkId int) (*TestConfig, error) {
 	var testConfig TestConfig
 
 	configData := string(static.MustAsset("config/test-data.json"))
 	if err := json.Unmarshal([]byte(configData), &testConfig); err != nil {
 		return nil, err
 	}
 
-	pass := os.Getenv(passphraseEnvName)
-	testConfig.Account1.Password = pass
-	testConfig.Account2.Password = pass
-	testConfig.Account3.Password = pass
+	if networkId == params.StatusChainNetworkID {
+		accountsData := string(static.MustAsset("config/status-chain-accounts.json"))
```

Comment:
This is maybe better because at least consistent but just FYI if you omit conversion to string, you can omit conversion to `[]byte` in the next line.